### ISSUE TITLE
Wrong health check for created VM

### ISSUE THREAD

**Issue by steled** [gomorpheus/terraform-provider-morpheus#58]
We created an instance layout and node type (see both below) for Ubuntu OS and that works quite well.

The problem we see is when we create an instance out of this layout the configuration
for the health check is wrong. The type oft he health check is Docker (but should be VM) and therefore
Morpheus can’t do the health checks and the availability is going down.

**node_type.tf**
```terraform
data "morpheus_virtual_image" "template_ubuntu2204" {
  name = "Template-Ubuntu2204"
}

data "morpheus_script_template" "tmout_setting" {
  name = "TMOUT setting"
}

resource "morpheus_node_type" "ubuntu_2204" {
  name             = "Ubuntu 22.04"
  short_name       = "ubuntu"
  technology       = "vmware"
  version          = "22.04"
  virtual_image_id = data.morpheus_virtual_image.template_ubuntu2204.id
  category         = "ubuntu"

  script_template_ids = [
    data.morpheus_script_template.tmout_setting.id
  ]

  service_port {
    name     = "SSH"
    port     = "22"
  }
}
```

**instance_layout.tf**
```terraform
data "morpheus_instance_type" "dev_ubuntu_lts" {
  name = "DEV - Ubuntu LTS"
}

data "morpheus_workflow" "dev_linux_workflow" {
  name = "DEV - Linux workflow"
}

data "morpheus_node_type" "ubuntu_2204" {
  name = "Ubuntu 22.04"

  depends_on = [
    morpheus_node_type.ubuntu_2204
  ]
}

resource "morpheus_instance_layout" "ubuntu_2204" {
  instance_type_id           = data.morpheus_instance_type.dev_ubuntu_lts.id
  name                       = "Ubuntu 22.04"
  version                    = "22.04"
  technology                 = "vmware"
  creatable                  = true
  minimum_memory             = 2048
  workflow_id                = data.morpheus_workflow.dev_linux_workflow.id

  node_type_ids = [
    data.morpheus_node_type.ubuntu_2204.id
  ]

  depends_on = [
    morpheus_node_type.ubuntu_2204
  ]
}
```

**Comment by martezr** [comment]
It looks like there is a value not being set in the API call that is not exposed via the UI that associates the VM with the appropriate check. We're looking to get a release out this week that address this issue and a few others.

**Comment by martezr** [comment]
This issue has been fixed in the latest release (0.8.0) of the provider - https://github.com/gomorpheus/terraform-provider-morpheus/blob/master/CHANGELOG.md#080-february-23-2023
